lib/ts/recipe/session/sessionClass.ts

```diff
@@ -164,4 +144,78 @@ export default class Session implements SessionContainerInterface {
         }
         return sessionInfo.expiry;
     };
+
+    assertClaims = async (claimValidators: SessionClaimValidator[], userContext?: any): Promise<void> => {
+        let validateClaimResponse = await this.helpers.getRecipeImpl().validateClaims({
+            accessTokenPayload: this.getAccessTokenPayload(userContext),
+            userId: this.getUserId(userContext),
+            claimValidators,
+            userContext,
+        });
+
+        if (validateClaimResponse.accessTokenPayloadUpdate !== undefined) {
+            await this.mergeIntoAccessTokenPayload(validateClaimResponse.accessTokenPayloadUpdate, userContext);
+        }
+
+        if (validateClaimResponse.invalidClaims.length !== 0) {
+            throw new STError({
+                type: "INVALID_CLAIMS",
+                message: "INVALID_CLAIMS",
+                payload: validateClaimResponse.invalidClaims,
+            });
+        }
+    };
+
+    fetchAndSetClaim = async <T>(claim: SessionClaim<T>, userContext?: any) => {
+        const update = await claim.build(this.getUserId(userContext), userContext);
+        return this.mergeIntoAccessTokenPayload(update, userContext);
+    };
+
+    setClaimValue = <T>(claim: SessionClaim<T>, value: T, userContext?: any) => {
+        const update = claim.addToPayload_internal({}, value, userContext);
+        return this.mergeIntoAccessTokenPayload(update, userContext);
+    };
+
+    getClaimValue = async <T>(claim: SessionClaim<T>, userContext?: any) => {
+        return claim.getValueFromPayload(await this.getAccessTokenPayload(userContext), userContext);
+    };
```

Comment:
```suggestion
    getClaimValue = async <T>(claim: SessionClaim<T>, maxAgeInSeconds?: number, userContext?: any) => {
        // if maxAgeInSeconds undefined, we set it to infinte
        // if value doesnt exist in paylaod, we call fetchAndSet, and then return it
        // if max age has passed, we call fetchAndSet and then return
        // else we just return
        return claim.getValueFromPayload(await this.getAccessTokenPayload(userContext), userContext);
    };
```

Reply:
Not doing now - will do later: https://github.com/supertokens/supertokens-node/issues/368